example/ion-sfu-cluster-load-tool/README.md

```diff
@@ -0,0 +1,80 @@
+## Ion-sfu load testing tool
```

Comment:
Please update the README to reflect the purpose of the example, it is not clear what this example does, or why it's different than the other load tool example.